tests/recipient_validation/test_phone_number.py

```diff
@@ -73,6 +71,11 @@
     "0845 46 31",  # short but not 01x or 08x
 ]
 
+premium_uk_landlines = [
+    "0845 46 46",  # short premium
+    "0900 123 4567",  # premium
```

Comment:
what about 0870, 0871, 0872 and 0873

Reply:
I can add some more test cases, these were the ones we were currently validating against